backend/authors/serializers.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+from rest_framework import serializers
+from .models import Author
+
+class AuthorSerializer(serializers.ModelSerializer):
+    type = serializers.SerializerMethodField(read_only=True)
+
```

Comment:
If you want to add followers to the author serializer you can do it by the following code:
`follower = serializers.SerializerMethodField(read_only=True)`
# then add 'follower' to the fields array below
```
def get_follower(self, obj):
    return FollowerSerializer(obj.follower.all(), many=True).data
```

Reply:
Fixed